astrocut/cube_cut.py

```diff
@@ -895,9 +900,8 @@ def cube_cut(
                 print("ymin,ymax: {}".format(self.cutout_lims[0]))
 
             # Make the cutout
-            img_cutout, uncert_cutout, aperture = self._get_cutout(
-                getattr(cube[1], cube_data_prop), threads=threads, verbose=verbose
-            )
+            img_cutout, uncert_cutout, aperture = self._get_cutout(getattr(cube[1], cube_data_prop), threads=threads,
+                                                                   verbose=verbose)
```

Comment:
we should probably align on what formatting whitespace formatting we're using on this project to avoid unnecessary changes back and forth...

I use [darker](https://github.com/akaihola/darker) which just applies black selectively to my set of changes (instead of on the entire file).